src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.html

```diff
@@ -126,6 +126,7 @@
     <ng-container>
       <mat-chip-row
         *ngFor="let item of _selectedOptions"
+        [hidden]="item.isHidden"
```

Comment:
didn't we already filter these out when creating _selectedOptions? Is this still needed here now?